sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Generated/Models/ServicesProperties.cs

```diff
@@ -46,7 +45,7 @@ public ServicesProperties()
         /// configuration of the service instance.</param>
         /// <param name="exportConfiguration">The settings for the export
         /// operation of the service instance.</param>
-        public ServicesProperties(IList<ServiceAccessPolicyEntry> accessPolicies, string provisioningState = default(string), ServiceCosmosDbConfigurationInfo cosmosDbConfiguration = default(ServiceCosmosDbConfigurationInfo), ServiceAuthenticationConfigurationInfo authenticationConfiguration = default(ServiceAuthenticationConfigurationInfo), ServiceCorsConfigurationInfo corsConfiguration = default(ServiceCorsConfigurationInfo), ServiceExportConfigurationInfo exportConfiguration = default(ServiceExportConfigurationInfo))
```

Comment:
The re-ordering of parameters here is not part of the new API version 2020-03-15; it must have been made as a change to the previous 2019-09-16 version.  This parameter re-ordering seems like a breaking change (?), in which case I suspect I should change the <Version> in the .csproj file to 2.0.0 (instead of 1.2.0, which is in the first iteration of this PR) from 1.1.0.  And, make corresponding changes to AssemblyInfo.cs file

Thoughts?

Reply:
Based on this, in comment 2 I'm changing the version to 2.0.0